ruddock/modules/users/routes.py

```diff
@@ -26,17 +27,64 @@ def show_memberlist(search_type):
       memberlist=memberlist,
       search_type=search_type)
 
-@blueprint.route('/view/<username>')
+@blueprint.route('/view/user_id=<user_id>')
```

Comment:
why not just view/<user_id>? the = makes it look like it's a paramter in a GET request